src/components/Filter/FilterActions.tsx

```diff
@@ -0,0 +1,109 @@
+import { Theme } from "@material-ui/core/styles";
+import TextField, { TextFieldProps } from "@material-ui/core/TextField";
+import { makeStyles } from "@material-ui/styles";
+import React from "react";
+
+import { FilterContentSubmitData, IFilter } from "../Filter";
+import Filter from "./Filter";
+
+const useInputStyles = makeStyles({
+  input: {
+    padding: "10px 12px"
+  },
+  root: {
+    flex: 1
+  }
+});
+
+const Search: React.FC<TextFieldProps> = props => {
+  const classes = useInputStyles({});
+  return (
+    <TextField
+      {...props}
+      className={classes.root}
+      inputProps={{
+        className: classes.input
+      }}
+    />
+  );
+};
+
+const useStyles = makeStyles(
+  (theme: Theme) => ({
+    actionContainer: {
+      display: "flex",
+      flexWrap: "wrap",
+      padding: `${theme.spacing.unit * 1.5}px ${theme.spacing.unit * 3}px ${
```

Comment:
You use one time `px` value one time `theme.spacing.unit`. We should set the correct pattern.